Orange/widgets/utils/webview.py

```diff
@@ -556,9 +556,9 @@ def _evalJS(self, code):
             if sip.isdeleted(self):
                 return
```

Comment:
Lint suggests that you return None here.

(Either all return statements in a function should return an expression, or none of them should.)